--- solcover/coverage.go
+++ solcover/coverage.go
@@ -5,7 +5,6 @@
 	"fmt"
 	"math/big"
 	"sort"
-	"time"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/vm"
@@ -148,7 +147,7 @@
 
 func (*coverageCollector) CaptureTxEnd(restGas uint64) {}
 
-func (*coverageCollector) CaptureEnd(output []byte, gasUsed uint64, t time.Duration, err error) {}
+func (*coverageCollector) CaptureEnd(output []byte, gasUsed uint64, err error) {}
 
 func (cc *coverageCollector) CaptureEnter(typ vm.OpCode, from common.Address, to common.Address, input []byte, gas uint64, value *big.Int) {
 	cc.contracts = append(cc.contracts, to)
